mod-entities-links/src/main/resources/spitfire/mod-entities-links/features/samples/links/createTwoLinks.json

```
@@ -10,6 +11,7 @@
       ]
     },
     {
+      "linkingRuleId": 7,
```

Should be strict with linking rules `/linking-rules/instance-authority`
```suggestion
      "linkingRuleId": 1,
```